ZeldaFullEditor/Gui/DungeonMain.cs

```diff
@@ -400,6 +401,9 @@ public void SaveToolStripMenuItem_Click(object sender, EventArgs e)
 
                 if (this.saveSettingsArr[8] && save.SaveEntrances(DungeonsData.Entrances, DungeonsData.StartingEntrances))
                 {
+                    // Set entrances expansion Bytes
+                    ROM.DATA[0x07F000] = 00;
```

Comment:
constant, think this is used in another file i already commented on so definitely should have one

Reply:
Done